syntax = "proto3";

package protos.dummy;

import "protos/dummy/dummy_0.proto";

message Dummy1One {
    string f1 = 1;
    int32 f2 = 2;
    double f3 = 3;
    protos.dummy.Dummy0One something = 4;
}

message Dummy1Two {
    Dummy1One dummy_one = 1;
    repeated int32 repeated_ints = 2;
}
